experiments/compositions/expanders/cel-expander/config/expanders/cel-v0.0.1.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cel-v0.0.1
+  namespace: system
+  labels:
+    control-plane: expander-cel-v0.0.1
```

Comment:
So does it mean we can only use expander-cel for versionv0.0.1 since it is hardcoded here?
if we want to use newer versions, do we need to change this field and the file name?

Reply:
ah good point. This is just a label. Needs to be fixed.